meson.build

```diff
@@ -1,9 +1,11 @@
 project('vala-language-server', 'vala', 'c')
 
 valac = meson.get_compiler('vala')
-libvala_version = run_command(valac, '--api-version').stdout().strip()
+# libvala_version = run_command(valac, '--api-version').stdout().strip()
+# 
+# libvala = dependency('libvala-@0@'.format(libvala_version))
 
-libvala = dependency('libvala-@0@'.format(libvala_version))
+libvala = dependency('libvala-0.48')
```

Comment:
why hardcode?

Reply:
Good point. What I really want here is a dependency version requirement for `libvala >= 0.48`

Reply:
The problem is that this is a little tricky since libvala includes the version in the package name.